server/config/passport.js

```diff
@@ -0,0 +1,31 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth20');
+const {Users} = require('../db/index').models;
+
+passport.serializeUser((user, done) => {
+  done(null, user);
+});
+
+passport.deserializeUser(async (user, done) => {
+  done(null, user);
+});
+
+passport.use(
+  new GoogleStrategy({
+    clientID:'307708163444-gr53a33uvr0ls2hfsronn6vdpkfkg555.apps.googleusercontent.com',
```

Comment:
pls put on .env on server side

DATABASE_URL=
GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=
SESSION_KEY=

Reply:
I made another commit and another push, did I fix it...?  LOL